preupg/application.py

```diff
@@ -346,12 +346,24 @@ def clean_scan(self):
     def prepare_for_generation(self):
         """Function prepares the XML file for conversion to HTML format"""
         for report in self._get_reports():
+            # at first, check which report style should be used.
```

Comment:
Do not use "at first" if you don't use "but" in the same sentence. Use "First" instead.

Reply:
Thanks :-)